src/uas/UAS.cc

```diff
@@ -2827,21 +2825,6 @@ void UAS::enableHilXPlane(bool enable)
         }
         qDebug() << "CREATED NEW XPLANE LINK";
         simulation = new QGCXPlaneLink(this);
-
-        float noise_scaler = 0.1f;

```

Comment:
I agree with this.  It would be nice to re-examine where the aerodynamics came from for the Rascal aircraft.  Ideally we could generate a wiki page about what each of aero parameters are currently, why they are that value, and how to change them for your individual setup if you need more realistic flight dynamics